tc/core/polyhedral/memory_promotion.cc

```diff
@@ -461,14 +461,20 @@ isl::multi_aff dropDummyTensorDimensions(
   space = add_range(space, list.n());
   return isl::multi_aff(space, list);
 }
+
+inline void unrollAllMembers(detail::ScheduleTreeElemBand* band) {
```

Comment:
Make a TODO that we want to investigate this in the context of long compilation times?